app/Jobs/CreateEmployeeJob.php

```
@@ -33,42 +28,13 @@ public function __construct(public array $employees, public $company_id, public
      */
     public function handle(): void
     {
-        $source = $this->source;
         $employees = $this->employees;
         try{
             DB::beginTransaction();
-        if($source == 'manual'){
-            $this->handleManualCreate($employees);
-        }else{
             $this->handleCsvCreate($employees);
-        }
-        DB::commit();
-    }catch(\Exception $e){
-        DB::rollback();
-    }
-    }
-
-    protected function handleManualCreate($employees){
-        foreach ($employees as $key => $employee) {
-           $user = User::create([
-                'email' => $employee['email'],
-                'password' => Hash::make($employee['email']),
-            ]);
-
-            Employee::create([
-                'user_id' => $user->id,
-                'company_id' => $this->company_id,
-                'first_name' => $employee['first_name'],
-                'last_name'  => $employee['last_name'],
-                'role'  => $employee['role'],
-                'staff_number' => $employee['staff_number'] ?? null,
-                'pay_rate' => $employee['pay_rate'],
-                'phone_number' => $employee['phone_number'],
-                'address' => $employee['address'] ?? null,
-                'zip_code' => $employee['zip_code'] ?? null,
-            ]);
-
-            EmployeeCreated::dispatch($user->refresh());
+            DB::commit();
+        }catch(\Exception $e){
```

can you please throw the exception so we know what went wrong `thow $e` after ` DB::rollback()`